test/test_insolation.jl

```diff
@@ -73,22 +64,3 @@ end
 F_NH = sort(F_arr[l_arr .>= 0])
 F_SH = sort(F_arr[l_arr .<= 0])
 @test F_NH ≈ F_SH rtol=rtol
-
-## Test invariance of zonal-mean insolation under rotation of ϖ
```

Comment:
This test was nice. Why not keeping it? (It doesn't seem to depend on the calendar conventions.)

Reply:
I removed it because I was planning to remove the parameter `lon_perihelion` from CLIMAParameters since it's no longer used. But I can keep the test and just change the `lon_perihelion_epoch` instead. That'll be the same thing.